src/css/style.scss

```diff
@@ -279,3 +285,32 @@ h1 {
 .blur {
   filter: blur(5px);
 }
+
+//form and pop-up-card
+.pop-up-card {
+  background-color: #9a9dab;
+  border: 2px solid black;
+  height: 20em;
+  margin-left: 3em;
+  position: absolute;
+  right: 200px;
```

Comment:
We should later figure out how to make the pop-up card stay a bit more in the middle as its a little skewed to the right. 
![image](https://user-images.githubusercontent.com/53014933/78623979-d03da980-7845-11ea-929b-02b626772ea5.png)

Reply:
Agreed. We tried a couple different things to flex it ... but the position: absolute (our best guess) kept leaving it unchanged. So we had to hard code it which means that in any different sized brower it looks awkward. Happy to try to figure a way around it... i wonder if grid would work, we did not try that approach.